lib/rspec/rails/example/channel_example_group.rb

```diff
@@ -15,7 +15,7 @@ module ClassMethods
   end
 end
 
-if ::Rails::VERSION::MAJOR >= 6
+if ::Rails::VERSION::MAJOR >= 6 && defined?(ActionCable)
```

Comment:
Maybe we could use instead `RSpec::Rails::FeatureCheck.has_action_cable_testing?`

https://github.com/rspec/rspec-rails/pull/2071/files#diff-bf13f1e6c52828697cb1c321683f5360R36

WDYT @palkan?

Reply:
> Maybe we could use instead RSpec::Rails::FeatureCheck.has_action_cable_testing?

I think, unifying checks is a good idea.
Though for some (maybe, historical) reason it hasn't been used in `rspec-rails` (I used Active Job integration as an example).

And we have `feature_check` required after example groups, probably, for a reason (I don't know about).

🤷‍♂️

Reply:
👍 for using a name feature flag